yellow-paper/docs/public-vm/avm.md

```diff
@@ -508,10 +422,10 @@ if !nestedContext.results.reverted AND instr.opcode != STATICCALL_OP:
     context.accruedSubstate.append(nestedContext.accruedSubstate)
 ```
 
-Regardless of whether a nested context has reverted, its journal updates are absorbed into the calling context along with a new `nestedCalls` entry.
+Regardless of whether a nested context has reverted, its [world state access trace](./state#world-state-access-trace) updates are absorbed into the calling context along with a new `contractCalls` entry.
 ```
-context.journal = nestedContext.journal
-context.journal.append(nestedContext.address, nestedContext.machineState.reverted)
+context.worldStateAccessTrace = nestedContext.worldStateAccessTrace
```

Comment:
is this replacing the journal?

Reply:
Nope, the trace is a circuit input. The journal handles staging modifications and basically is an interface to Aztec state. The journal (probably) will output the trace to be handed off to witgen/circuit.